light-zk.js/src/account.ts

```diff
@@ -35,7 +34,8 @@ export class Account {
   };
   eddsa: any;
   aesSecret?: Uint8Array;
-
+  hashingSecret?: Uint8Array;
+  prefixHash?: Uint8Array;
```

Comment:
what's your reasoning for saving a prefix hash here?

Reply:
I mimic-ed the `aesSecret` scope/meaning/behaviour, you've better context if you are convinced that's makes no sense I'll remove it

Reply:
after code inspection I agree with you, removed. made no sense to have it